techeria_controller/techeria_app/views.py

```diff
@@ -71,43 +71,144 @@ def product(request):
 
 
 
-        if category == "Buyer":
-            buyer.user_name = user_name
-            buyer.first_name=first_name
-            buyer.last_name=last_name
-            buyer.user_name=user_name
-            buyer.email=email;
-            buyer.date_of_birth=date_of_birth
-            buyer.mobile_number=mobile_number
-            buyer.address=address
-            buyer.city=city
-            buyer.state=state
-            buyer.zip_code=zip_code
-            buyer.country=country
-            user = User.objects.create_user(username=user_name, password=password, email=email, first_name=first_name, last_name=last_name)
-            buyer.save()
-            user.save()
-            return redirect("/")
+        from django.db.models.fields import NullBooleanField
+from django.http import response
+from django.shortcuts import redirect, render
+from techeria_app.models import BuyerModel, SellerModel, Products
+from django.contrib.auth.models import User, auth
+from django.contrib import messages
 
-        elif category == "Seller":
-            seller.user_name = user_name
-            seller.first_name=first_name
-            seller.last_name=last_name
```

Comment:
Did you set the verification email?

Reply:
ok, I will add the verification. When users want to sign up, the verification email will be sent and after sending the verification only the users will be able to sign up.